sycl/source/detail/queue_impl.hpp

```diff
@@ -1000,6 +1013,14 @@ class queue_impl {
   // recording commands to it.
   std::weak_ptr<ext::oneapi::experimental::detail::graph_impl> MGraph{};
 
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
+  // CMPLRLLVM-66082
+  // This member should be part of the sycl::interop_handle class, but it
+  // in an API breaking change. So member lives here temporarily where it can
```

Comment:
```suggestion
  // This member should be part of the sycl::interop_handle class, but it
  // is an API breaking change. So member lives here temporarily where it can
```